src/core/sys/posix/stdio.d

```diff
@@ -217,12 +273,16 @@ else version( DragonFlyBSD )                 // for DragonFlyBSD
     version = HaveMemstream;
 else version( OpenBSD )                      // as of OpenBSD 5.4
     version = HaveMemstream;
+else version( CRuntime_UClibc )
+    version = HaveMemstream;
 
 version( HaveMemstream )
 {
     FILE*  fmemopen(in void* buf, in size_t size, in char* mode);
     FILE*  open_memstream(char** ptr, size_t* sizeloc);
-    FILE*  open_wmemstream(wchar_t** ptr, size_t* sizeloc);
+    version( CRuntime_UClibc ) {}
+    else
```

Comment:
Can just put the `else` at the end of the line above and not re-indent the next line.